app/lib/config/providers/ElementTemplatesProvider.js

```diff
@@ -145,6 +145,7 @@ function getTemplatesForPath(path) {
  */
 function globTemplates(path) {
   return globFiles('element-templates/**/*.json', {
-    cwd: path
+    cwd: path,
+    dot: true
```

Comment:
What does this do?

Reply:
Hidden files support: https://www.npmjs.com/package/fast-glob#dot